### ISSUE TITLE
Basic sample @gitbeaker/node throws error, asking for `Requester`

### ISSUE THREAD

**Issue by chez14** [jdalrymple/gitbeaker#885]
**Description**

Basic sample code didn't work, it ask for `Requester` on `@gitbeaker/node` as it throws an error:
```
(node:27143) UnhandledPromiseRejectionWarning: ReferenceError: Requester must be passed
    at t.s (/home/hayashi/workspace/nyanko-bot/node_modules/@gitbeaker/core/dist/index.js:1:622)
    at new t (/home/hayashi/workspace/nyanko-bot/node_modules/@gitbeaker/core/dist/index.js:1:20633)
    at /home/hayashi/workspace/nyanko-bot/node_modules/@gitbeaker/core/dist/index.js:1:3638
    at Array.forEach (<anonymous>)
    at e.<anonymous> (/home/hayashi/workspace/nyanko-bot/node_modules/@gitbeaker/core/dist/index.js:1:3585)
    at new e (/home/hayashi/workspace/nyanko-bot/node_modules/@gitbeaker/requester-utils/dist/index.js:1:782)
    at Object.<anonymous> (/home/hayashi/workspace/nyanko-bot/src/bot/gitlab.ts:20:19)
    at step (/home/hayashi/workspace/nyanko-bot/src/bot/gitlab.ts:44:23)
    at Object.next (/home/hayashi/workspace/nyanko-bot/src/bot/gitlab.ts:25:53)
    at fulfilled (/home/hayashi/workspace/nyanko-bot/src/bot/gitlab.ts:16:58)
```

**Steps to reproduce**
```ts
	import { UsersBundle } from "@gitbeaker/node";
	let glUsers = new UsersBundle({
		token: "fake-token"
	});
	let userInfo: any = await glUsers.Users.current();
```

**Expected behaviour**
It didn't throw error

**Actual behaviour**
Throws error, asking for Requester

**Possible fixes**
Reinstall to `v19.7.1` fix the problem.

**Comment by chez14** [comment]
Dev env info:

### `package.json`
```json
{
  "dependencies": {
    "@gitbeaker/node": "^21.0.1",
    "@types/node": "^14.0.13",
    "cookie-parser": "~1.4.4",
    "debug": "~2.6.9",
    "discord.js": "^12.2.0",
    "dotenv": "^8.2.0",
    "express": "~4.16.1",
    "match-all": "^1.2.5",
    "morgan": "~1.9.1",
    "mysql": "^2.18.1",
    "typeorm": "^0.2.25",
    "typescript": "^3.9.5"
  },
  "devDependencies": {
    "nodemon": "^2.0.4",
    "ts-node": "^8.10.2"
  },
}
```

Node:
```
nyanko-bot on  master [!?] is 📦 v0.0.0 via ⬢ v12.18.0 
❯ node -v && npm -v && yarn -v
v12.18.0
6.14.4
1.19.1
```

**Comment by chez14** [comment]
Update:`v19.7.1` works as expected. It brokes on `v21` (i didn't try the `v20` one though).

**Comment by jdalrymple** [comment]
@chez14 :facepalm: my bad, ill fix this up today

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`21.3.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/21.3.0) :rocket:

**Comment by jdalrymple** [comment]
Should be good now!
